alacritty_terminal/src/term/mod.rs

```diff
@@ -1840,8 +1850,8 @@ impl<T: EventListener> Handler for Term<T> {
         trace!("Clearing screen: {:?}", mode);
         let template = self.grid.cursor.template;
 
-        // Remove active selections.
-        self.selection = None;
+        let num_lines = self.grid.num_lines().0 - 1;
```

Comment:
```suggestion
        let last_line = self.num_lines().0 - 1;
```

We shouldn't name things `num_lines` when we subtract one to represent the last line.

Alternatively `last_visible_line` would be even more clear.